README.md

```diff
@@ -28,6 +28,10 @@ bash alias dcp="docker-compose -f docker-compose-production.yml"
 
 #### Готовим код к работе
 ```bash
+mkdir fidals && cd fidals
+# clone outer dependency repository
+git clone git@github.com:fidals/refarm-site.git
+# clone shopelectro repository
```

Comment:
It's redundant comment, plz do it. We avoid obvious comments because just several ones will lead every comment to dev guy's bullshit filter =)